lib/src/ast/event.dart

```diff
@@ -159,7 +158,7 @@ class ParsedEventAst extends TemplateAst
   @override
   String get postfix {
     final split = _nameWithoutParentheses.split('.');
-    return split.length > 1 ? split.last : null;
+    return split.length > 1 ? split[1] : null;
```

Comment:
Oh wait I'm already throwing it haha. It's in Recursive. Look for 'PROPERTY_NAME_TOO_MANY_FIXES' and 'EVENT_NAME_TOO_MANY_FIXES'.